run

```diff
@@ -521,6 +529,10 @@ run_mkdrive() {
             local targetdir=$(realpath $5)
             local targetvol=(-v "$targetdir:$targetdir")
         fi
+        docker run --rm                        \
+            -v /lib/modules:/lib/modules       \
+            "$DOCKER_IMAGE"                    \
+            modprobe loop
```

Comment:
I wonder if modprobe is going to work at all in a non-privileged container?

Reply:
Yes, but only on Mac OS.
I remove load_loop_module and run docker as privileged on both platform.